docs/source/pyplots/section/section_lexicon.py

```diff
@@ -0,0 +1,13 @@
+import matplotlib.pyplot as plt
+
+import deltametrics as dm
+
+rcm8cube = dm.sample_data.cube.rcm8()
+rcm8cube.register_section('demo', dm.section.StrikeSection(y=10))
+
```

Comment:
This is a nice example. Functionality-wise, I like the loading of the cube and then the slicing of the section.

When the `rcm8cube` object is created it is basically loading (via the `io` wrapper etc) the NetCDF4 file. So the *variables* available can be listed using `rcm8cube.variables`. However once we move to the section `rcm8cube.sections`, we lose the ability to read the variables available in the section. Is the list of variables something worth passing along to the section? Unless you know to check `rcm8cube.variables`, you don't know that in addition to 'strata_sand_frac', you could call 'velocity' when visualizing the section.
 
A lot of the I/O and variable handing seems to have been done carefully with respect to name-spaces and memory, but my ham-handed suggestion is something like `self[name].variables = self.variables` in the `register_section` function

Reply:
yep, definitely a good idea. I'll implement something like this.